### PR TITLE
Allow hemiplegic users to pick which side is disabled

### PR DESCRIPTION
**Pull request by 00-Steven**
## About The Pull Request

Broadly this code is copied over from the preference code used for picking prosthetic limbs, with a few changes to make it fit.
It also changes how the quirk displays in the medical records, now mentioning which side is disabled instead of just half their body.
## Why It's Good For The Game

So of course, first and foremost, I want it personally.
I'm now running a hemiplegic character on Manuel, and I want to able to pick which side is hemiplegic to match me using heterochromatic eyes to make it look like they're missing an eye. It's minor, but I think it'd be nice.

Then, because of that, I think it'd be nice to have for posterity. If I want it, and I can code it, then the next person who wants it doesn't have to, y'know?

Lastly, gameplay-wise, it doesn't really matter which side is hemiplegic, you're permanently missing an arm and a leg either way. So I feel there's practically no 'powergaming concerns' to be had when it's this utter shitass to use regardless and the benefits to be gained would be infinitesimally small in comparison.
## Changelog
:cl:
add: Hemiplegic now lets you pick which side is disabled.
add: The medical records text for hemiplegics now shows which side is disabled.
/:cl:

### CODE REVIEW

File: code/datums/quirks/negative_quirks/hemiplegic.dm [hunk 1]
```diff
@@ -12,10 +12,20 @@
...
+	var/datum/brain_trauma/severe/paralysis/hemiplegic/side_choice = GLOB.side_choice_hemiplegic[client_source?.prefs?.read_preference(/datum/preference/choiced/hemiplegic)]
+	if(isnull(side_choice))  // Client gone or they chose a random side
+		side_choice = GLOB.side_choice_hemiplegic[pick(GLOB.side_choice_hemiplegic)]
+
 	var/mob/living/carbon/human/human_holder = quirk_holder
-	var/trauma_type = pick(/datum/brain_trauma/severe/paralysis/hemiplegic/left, /datum/brain_trauma/severe/paralysis/hemiplegic/right)
-	human_holder.gain_trauma(trauma_type, TRAUMA_RESILIENCE_ABSOLUTE)
+	var/side_string = "[side_choice::paralysis_type]"
+
+	medical_record_text = "Patient has an untreatable impairment in motor function on the [side_string] half of their body."
```
Comment (san7890) [inline code/datums/quirks/negative_quirks/hemiplegic.dm:27]:
```suggestion

	medical_record_text = "Patient has an untreatable impairment in motor function on the [side_choice::paralysis_type] half of their body."
```

no real need to have the variable there

**Review by san7890 — CHANGES_REQUESTED** [review]

File: code/datums/quirks/negative_quirks/hemiplegic.dm (hunk 1 above)
Comment (00-Steven) [reply code/datums/quirks/negative_quirks/hemiplegic.dm:27]: oh yeah totally right

**Comment by 00-Steven** [comment]
wh, the fuck
image
no info for me
never seen it do that before

**Comment by san7890** [comment]
freak accident

**Pull request merged**